src/CSV.jl

```diff
@@ -1,45 +1,46 @@
 module CSV
 
-using Mmap, Dates, Random, Unicode, Parsers, Tables, CategoricalArrays, WeakRefStrings
+using Mmap, Dates, Random, Unicode, Parsers, Tables, CategoricalArrays, PooledArrays, WeakRefStrings, DataFrames
 
-newline(code::Parsers.ReturnCode) = (code & Parsers.NEWLINE) > 0
-
-const CatStr = CategoricalString{UInt32}
-
-struct Error <: Exception
-    error::Parsers.Error
-    row::Int
-    col::Int
+function validate(fullpath::Union{AbstractString,IO}; kwargs...)
+    Base.depwarn("`CSV.validate` is deprecated. `CSV.read` now prints warnings on misshapen files.", :validate)
+    Tables.columns(File(fullpath; kwargs...))
+    return
 end
 
-function Base.showerror(io::IO, e::Error)
-    println(io, "CSV.Error on row=$(e.row), column=$(e.col):")
-    showerror(io, e.error)
-end
+import Tables.transform
+export transform
```

Comment:
We already export it, so it'd be breaking if we stopped. But maybe we should make that change anyway, since it's a pretty common name and could easily collide out in the wild.